package.json

```
@@ -30,7 +30,10 @@
     "changeset-version": "pnpm changeset version && pnpm style:fix"
   },
   "dependencies": {
-    "buffer": "^6.0.3"
+    "abort-controller": "^3.0.0",
+    "buffer": "^6.0.3",
+    "node-fetch": "2",
+    "pako": "^2.1.0"
```

Recent React Native and Node versions have built-in support for abort-controller and fetch, so it shouldn't be needed to add this (node-fetch also won't work in react native). 

```suggestion
    "buffer": "^6.0.3",
    "pako": "^2.1.0"
```

---

Hm, but "pnpm build" seems to be breaking without these. I think it also check if we can use it in the browser environments

---

That's more a type issue. We shouldn't add these dependencies. The environment needs to provide these, but for the building we can add `"DOM"` to the `"lib"` in `tsconfig.json`